flink-staging/flink-streaming/flink-streaming-scala/src/test/java/org/apache/flink/streaming/scala/api/CsvOutputFormatITCase.java

```diff
@@ -36,6 +37,10 @@ protected void preSubmit() throws Exception {
 	@Override
 	protected void testProgram() throws Exception {
 		OutputFormatTestPrograms.wordCountToCsv(WordCountData.TEXT, resultPath);
+		OutputFormatTestPrograms.wordCountToCsv(WordCountData.TEXT, resultPath, 1);
```

Comment:
The `writeMode` by setting default is already exist.
It will check whether the file can be overwrite or not in `FileOutputFormat.java`:
`private static final void initDefaultsFromConfiguration(Configuration configuration) {
		final boolean overwrite = configuration.getBoolean(ConfigConstants
						.FILESYSTEM_DEFAULT_OVERWRITE_KEY,
				ConfigConstants.DEFAULT_FILESYSTEM_OVERWRITE);
	
		DEFAULT_WRITE_MODE = overwrite ? WriteMode.OVERWRITE : WriteMode.NO_OVERWRITE;
		
		final boolean alwaysCreateDirectory = configuration.getBoolean(ConfigConstants
						.FILESYSTEM_OUTPUT_ALWAYS_CREATE_DIRECTORY_KEY,
			ConfigConstants.DEFAULT_FILESYSTEM_ALWAYS_CREATE_DIRECTORY);
	
		DEFAULT_OUTPUT_DIRECTORY_MODE = alwaysCreateDirectory ? OutputDirectoryMode.ALWAYS : OutputDirectoryMode.PARONLY;
	}`